src/rt/test/harness.h

```diff
@@ -51,17 +51,17 @@ class SystematicTestHarness
 
     if (seed_upper < seed_lower)
     {
-      std::cout << "Seed_upper " << seed_upper << " seed_lower " << seed_lower
-                << std::endl;
+      Systematic::cout() << "Seed_upper " << seed_upper << " seed_lower "
```

Comment:
I am not sure I agree with pushing the command line messages only into the systematic logging.  These are really about how to run the application.

Reply:
You are right. I reverted all changes to harness.h as it does not involve the runtime just specific tests.